A1-http-connect-proxy-support.md

```diff
@@ -1,222 +1,242 @@
-HTTP CONNECT Proxy Support
---------------------------
-* Author(s): Mark D. Roth (roth@google.com), Eric Anderson (ejona@google.com)
-* Approver: a11r
-* Status: Implemented
-* Implemented in: C-core (Java and Go are in progress)
-* Last updated: 2017-03-20
-* Discussion at: https://groups.google.com/d/topic/grpc-io/zG7iYpDw6lk/discussion
+## HTTP CONNECT Proxy Support
+
+*   Author(s): Mark D. Roth (roth@google.com), Eric Anderson (ejona@google.com)
+*   Approver: a11r
+*   Status: Implemented
+*   Implemented in: C-core (Java and Go are in progress)
+*   Last updated: 2017-03-20
+*   Discussion at:
+    https://groups.google.com/d/topic/grpc-io/zG7iYpDw6lk/discussion
```

Comment:
These 8/9 lines are just source formatting change and does not fix anything AFAICS. Is that caused by the tooling?

Reply:
Yes